backend/project_alpha/api/tests/test_models.py

```diff
@@ -0,0 +1,51 @@
+from rest_framework.test import APITestCase
+from django.core.exceptions import ValidationError
+from project_alpha.web.models import User
+
+
+class UserModelTestCase(APITestCase):
+
+    def setUp(self):
+        self.username = 'test@example.com'
+        self.user = User.objects.create_user(self.username)
+
+    def test_happy_patch_all_validators_are_not_called(self):
+        result = self.user.set_password(raw_password='TestPassword123')
+        self.assertEqual(result, None)
+
+    def test_happy_patch_1_validator_failed_to_check(self):
+        """
+        Only 1 validator pass allowed
```

Comment:
Allowed to *not* pass only 1 validator. 
In addition, please specify in comments which validators in this case will pass and which aren't.